wp-content/themes/theme_name/inc/media.php

```diff
@@ -14,6 +14,15 @@
    */
   function cc_mime_types( $mimes ) {
     $mimes['svg'] = 'image/svg+xml';
+    // unset($mimes['doc']);
+    // unset($mimes['docx']);
+    // unset($mimes['xls']);
+    // unset($mimes['xlsx']);
+    // unset($mimes['psd']);
+    // unset($mimes['ppt']);
+    // unset($mimes['pptx']);
+    // unset($mimes['pps']);
+    // unset($mimes['ppsx']);
     return $mimes;
```

Comment:
Adding a filter would be good if we ever want to add a new mime type. That way we can hook to it in the plugin and add it.